cmdstatus.go

```diff
@@ -9,7 +9,14 @@ import (
 
 var statusCommand = &Command{
 	Run:       cmdStatus,
+	Short:     "print the current hack status of a module",
 	UsageLine: "status [module...]",
+	Long: `
+The status command prints the status of
+all modules that are currently replaced by local
```

Comment:
Isn't this the status of all modules that are replaced by a `gohack` target?

Reply:
Currently gohack doesn't distinguish between directory replace statements that it has added and any other directory replace statements. I'm not sure whether it should, tbh. One thing it probably *should* distinguish is between directories within the repo and directories outside it. Something for another PR.